src/Microsoft.HttpRepl/Commands/SetBaseCommand.cs

```diff
@@ -54,45 +54,29 @@ public async Task ExecuteAsync(IShellState shellState, HttpState state, ICorePar
                 catch { }
             }
 
-            if (state.BaseAddress == null || !Uri.TryCreate(state.BaseAddress, "swagger.json", out Uri result))
+            if (state.BaseAddress == null)
             {
-                state.Structure = null;
+                state.ApiDefinition = null;
             }
             else
             {
-                await SetSwaggerCommand.CreateDirectoryStructureForSwaggerEndpointAsync(shellState, state, result, cancellationToken).ConfigureAwait(false);
-                if (state.Structure != null)
-                {
-                    shellState.ConsoleManager.WriteLine(Strings.SetBaseCommand_SwaggerMetadataUriLocation + result);
-                }
-                else
+                string[] swaggerSearchPaths = new[] { "swagger.json", "swagger/v1/swagger.json" };
```

Comment:
should we pull these up into a static readonly field on the object?  Seems hard to find this magic list embedded this deeply in the code.

Reply:
I made this change for now, but I'm expecting this will get replaced with a setting eventually, so the lookup of the setting will be back down where that list was - that's why I left it there in the first place. But who knows when that'll happen, so... done.